ansible/inventories/production/opensrp-stack/group_vars/all/vars.yml

```diff
@@ -75,3 +75,7 @@ reveal_new_db: "192.168.100.252"
 opensrp_rabbitmq_host: "localhost"
 opensrp_rabbitmq_username: "opensrp_reveal"
 opensrp_rabbitmq_password: "{{ vault_opensrp_rabbitmq_password }}"
+
+reveal_ssl_dir: "../Reveal-Thailand/ansible/inventories/preview/opensrp-stack/files/ssl"
```

Comment:
Shouldn't this point to production certs?

Reply:
yeah good catch, some lazy copy paste